sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_usage_v2/metadata.yaml

```
@@ -8,6 +8,10 @@ labels:
   schedule: daily
 scheduling:
   dag_name: bqetl_monitoring
+  referenced_tables:
+  - - 'moz-fx-data-shared-prod'
+    - 'monitoring_derived'
+    - 'job_by_organization_v1'
```

Based on the diffs CI added to the PR it doesn't look like the DAG was updated with the task dependency that's needed.  After looking at the bqetl code some more it looks like `referenced_tables` is just used for generated documentation, and `depends_on` is intended for manually specifying task dependencies. However, there's what I'd consider a bug in the current DAG generation code where using `depends_on` always generates an external task sensor even if the task referenced is in the same DAG, which isn't appropriate for this use case.  So as a hack workaround until that bug is fixed you can manually set `upstream_dependencies` (which is meant to be auto-generated).  I've tested this locally and it does add the task dependency as expected.
```suggestion
  referenced_tables:
  - - 'moz-fx-data-shared-prod'
    - 'monitoring_derived'
    - 'job_by_organization_v1'
  upstream_dependencies:
  - task_id: monitoring_derived__jobs_by_organization__v1
    dag_name: bqetl_monitoring
```